src/js/controller/controller.js

```diff
@@ -183,6 +184,15 @@ define([
                 });
             });
 
+            _model.on('change:viewable', function(model, viewable) {
```

Comment:
You'll need to wrap this in the autostart logic one we're always observing the player container.